src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/GenericWhitespaceCheck.java

```diff
@@ -71,10 +77,25 @@
  * MyClass obj = new &lt;String&gt;MyClass();
  * </pre>
  * <p>
- * To configure the check:
+ * Examples with incorrect spacing:
  * </p>
  * <pre>
- * &lt;module name=&quot;GenericWhitespace&quot;/&gt;
+ * // violation, "&lt;" precedes a method name and is preceded with whitespace
+ * Set&lt;String&gt; s2 = Collections. &lt;String&gt;unmodifiableSet(s1);
+ * // violation, "&lt;" preceded with whitespace when following type name
+ * Queue &lt;Integer&gt; a;
+ * // violation, "&lt;" followed with whitespace
+ * LinkedList&lt; String&gt; l;
+ *
+ * // violation, "&gt;" preceded with whitespace
+ * Pair&lt;Integer, Integer &gt; p;
+ * // violation, "&gt;" not followed with whitespace
+ * HashMap&lt;Integer, String&gt;h;
+ * // violation, "&gt;" followed with whitespace when preceding method name
+ * Set&lt;Number&gt; s2 = Collections.&lt;Number&gt; synchronizedSet(s1);
+ *
+ * // violation under MethodParamPad, "&gt;" followed with whitespace in diamond operator
```

Comment:
`under MethodParamPad, ` user does not need this

Reply:
Removed that part and put the example together with the rest of the `>` violations